bedrock/mozorg/templates/mozorg/internet-health/privacy-security.html

```diff
@@ -452,9 +452,15 @@ <h3 class="topic-action-title">{{ _('What Mozilla is doing about government surv
       <h2>{{ _('Hungry for more?') }}</h2>
       <p class="report-cta">
         <span>
-        {% trans healthreport='https://internethealthreport.org' %}
-          Read on about Digital Inclusion in our <a href="{{ healthreport }}">Internet Health Report</a>.
-        {% endtrans %}
+        {% if l10n_has_tag('internet_health_subpage_rebrand') %}
+          {% trans healthreport='https://internethealthreport.org' %}
+            Read on about Digital Inclusion in our <a href="{{ healthreport }}">Internet Health Report</a>.
```

Comment:
Correct, this shouldn't have changed. I was sloppy. We should revert this to the original string.